docs/tutorials/notebooks/BIBD.ipynb

```diff
@@ -0,0 +1,912 @@
+      ],
+      "metadata": {
+        "id": "2al5-kz9JjdA"
+      }
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "_X4PB87CIyz4",
+        "outputId": "80c19416-ac53-4898-93a7-a61e68c02aea"
+      },
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "{}"
+            ]
+          },
+          "metadata": {},
+          "execution_count": 3
+        }
+      ],
+      "source": [
+        "%%conjure\n",
+        "letting crops be new type enum {🥔,🌽,🥦,🥕,🥒, 🍅}\n",
+        "letting farms be 4\n",
+        "letting crops_per_farm be 3\n",
+        "letting farms_per_crop be 2\n",
+        "letting overlap be 1"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The model will be defined in a new cell. It should start by accessing the provided parameters, this uses the given keyword, followed by the names of the parameters and their type."
+      ],
+      "metadata": {
+        "id": "DXAdjZdyJ1BA"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "`given farms, crops_per_farm, farms_per_crop, overlap: int`\n",
+        "\n",
+        "`given crops new type enum`"
+      ],
+      "metadata": {
+        "id": "tifxXJkAJ7sH"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Next, we need to define what we are looking for. The ‘find’ keyword indicates that the solver should find a value to for that variable. We want to find a set containing sets of crops. Each set of crops is a crop assignment for a farm."
+      ],
+      "metadata": {
+        "id": "F6pOOUzpKEHU"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "`given farms, crops_per_farm, farms_per_crop, overlap: int`\n",
+        "\n",
+        "`given crops new type enum`\n",
+        "\n",
+        "`find crop_assignment: set of set of crops`"
+      ],
+      "metadata": {
+        "id": "54TpE37nKF5A"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "%%conjure\n",
+        "find crop_assignment: set of set of crops"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "kG-nX4l8KzPt",
+        "outputId": "f5f874d0-88f6-401e-8ccb-6c2e0eaaf0ad"
+      },
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "{'crop_assignment': []}"
+            ]
+          },
+          "metadata": {},
+          "execution_count": 4
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "With no constraints it produces an empty set for crop assignment."
```

Comment:
Maybe "with no constraints Conjure produces an empty set for the crop assignment"?